parasoft-findings-teamcity-agent/src/main/resources/com/parasoft/findings/teamcity/agent/sa-pmd.xsl

```diff
@@ -110,6 +111,15 @@
         </xsl:element>
     </xsl:template>
 
+    <xsl:template match="ElDesc">
+        <xsl:text>&#xa;        - </xsl:text>
+        <xsl:call-template name="substring-after-last">
+            <xsl:with-param name="string" select="@srcRngFile"/>
+            <xsl:with-param name="delimiter" select="'/'"/>
```

Comment:
hi @yshi-parasoft , i've noticed in different report, the file's path looks different, e.g:  
- **srcRngFile="/Jtest Example/examples\bugdetective\InefficientCollectionRemoval.java"** 
- **srcRngFile="/com.parasoft:parabank/src/main/java/com/parasoft/bookstore/BookStoreDB.java"**
  it hard to decide use which delimiter to substring and i've try to check and substring with both delimiter while transforming. but it seems have some performence issue. maybe we can have discuss about this.

Reply:
Did you try to use `translate` to change all forward slashes to backward slashes before calling the template?
I just tried and it doesn't look like there is performance issue.

Reply:
I did't noticed this funtion. I've tried to use this function and it works well. thank you.